src/test/java/com/researchspace/webapp/integrations/fieldmark/FieldmarkMVCIT.java

```diff
@@ -0,0 +1,146 @@
+package com.researchspace.webapp.integrations.fieldmark;
+
+import com.researchspace.api.v1.controller.API_MVC_TestBase;
+import com.researchspace.model.User;
+import com.researchspace.service.UserConnectionManager;
+import com.researchspace.webapp.integrations.fieldmark.model.FieldmarkJsonExport;
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.Enumeration;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipFile;
+import org.apache.commons.io.FileUtils;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.web.WebAppConfiguration;
+import org.springframework.web.client.RestTemplate;
+
+@WebAppConfiguration
+public class FieldmarkMVCIT extends API_MVC_TestBase {
+
+  private @Autowired UserConnectionManager userConnectionManager;
+  private User user;
```

Comment:
It doesn't seem like these 2 fields, or the user init on line 35 are needed.

Reply:
thanks, removed